bin/i18n/resources/animations.rb

```diff
@@ -0,0 +1,27 @@
+require 'fileutils'
+require 'json'
+
+require_relative '../../animation_assets/manifest_builder'
+require_relative '../i18n_script_utils'
+
+module I18n
+  module Resources
+    module Animations
+      CROWDIN_FILE_PATH = "#{I18N_SOURCE_DIR}/animations/spritelab_animation_library.json".freeze
+
+      module_function
+
+      def sync_in
+        prepare
+      end
+
+      def prepare
+        FileUtils.mkdir_p(File.dirname(CROWDIN_FILE_PATH))
+
+        animation_strings = ManifestBuilder.new({spritelab: true, quiet: true}).get_animation_strings
+
+        File.write(CROWDIN_FILE_PATH, JSON.pretty_generate(animation_strings))
+      end
```

Comment:
Since the `sync_in` process can contain several operations for some resources, it would be nice to be consistent in this.

https://github.com/code-dot-org/code-dot-org/pull/52577/files#diff-ab2bee7f2a507471b23a9dce07d7b5ab6a336fa72fb50e17654d0b167264b08eR12-R15

Reply:
I believe we went with the `prepare` name because we have been using `# Preparing blah content` in the sync-in.
Some modules will content redact/restore method,  and later on when we add the sync-out,  a `distribute` method.

Reply:
The idea is to create a consistent template for future features to be added to the sync (self-service)